src/html/page.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" href="mystyle.css">

```

Comment:
From what I can tell, the CSS between the start screen and the "session" page is duplicated across two files: `home.css` and `mystyle.css`. This includes duplicate blocks of style code, like `.sidebar`.

Please just merge these into a single CSS file, which will make development easier.